### DISCUSSION TITLE
Disabling LEDs on rPi4b using HAOS. HELP!

### DISCUSSION THREAD

**Discussion by nn0de** [home-assistant/operating-system#3157] [Q&A]
So I just installed HAOS on one of my spare Raspberry Pi to try it out but I can't seem to locate a config.txt file turn turn off the LEDs on my Raspberry Pi. I can't seem to locate a /boot directory and I have seen someone else say it's located under /mnt but there is nothing inside my /mnt directory. I have created these directories in these locations and of course, nothing happens. Could someone please point me in the direction where I can put this prompt to disable the LEDs on my pi?

#Disable Power LED (Red)
dtparam=pwr_led_activelow=off
#Disable Activity LED (Green)
dtparam=act_led_trigger=none
dtparam=act_led_activelow=off
#Disable LAN LEDs
dtparam=eth_led0=14
dtparam=eth_led1=14

This is what i use on a regular raspberry pi OS install inside the /boot/config.txt file. I just cant locate a boot/config.txt file in HAOS.

**Comment by sairon** [comment]
You are likely using the SSH add-on, which runs a container with its own filesystem and has this folder unavailable. If you connect a display and keyboard, you can find the content of the boot partition in `/mnt/boot`. Since the boot partition uses a FAT32 filesystem, you can also modify it from any computer, just put the card into a card reader and you should see the content of it.

**Reply by jksmurf** [reply]
Thanks that is exactly what I am doing and I had the same issue as the OP i.e. no mnt/boot dir or config.txt file via SSH (Putty to SSH Server Addon using a Key), which was puzzling.

I'm surprised there isn't an Addon (maybe I missed it?) or an option in the System/Hardware to do this without having to resort to SSH or modifying it via a WinPC. Disabling the LEDs isn't really a security issue surely.
